tests/TestCase/Collection/CollectionTest.php

```diff
@@ -60,6 +60,60 @@ public function testArrayIsWrapped()
     }
 
     /**
+     * Tests that it is possible to convert an array into a collection
+     *
+     * @return void
+     */
+    public function testAvg()
+    {
+        $items = [1, 2, 3];
```

Comment:
Take a look at the other tests in this file. Use a `@dataProvider` so you can automatically test your function when both passing an array and passing a generator.